packages/splits-sdk/src/utils/balances.ts

```diff
@@ -111,7 +111,6 @@ const processBalanceMulticallResponse: (
     if (token === ADDRESS_ZERO) {
       balances[ADDRESS_ZERO] = balance
     } else {
-      if (!tokenData[token]) return // Unable to fetch token data
```

Comment:
this seems wrong. i assume you just need to pull in master or something?

Reply:
Yeah I'll merge main in again